src/main/java/org/kitteh/irc/client/library/implementation/ClientBuilder.java

```diff
@@ -126,6 +127,13 @@ public ClientBuilder realName(@Nonnull String name) {
 
     @Nonnull
     @Override
+    public ClientBuilder defaultingOutboundMessageMap(@Nonnull DefaultingOutboundMessageMap outboundMessageMap) {
```

Comment:
Yeah, it's supposed to always be set. Passing null to this doesn't make any sense, and would yield some NPE's

```java
    @Override
    public void shutdown() {
        this.shutdownInternal(this.defaultMessageMap.getDefault(DefaultMessage.QUIT).orElse(null));
    }
```